site/en/posts/windows-games-on-chromeos.md

```diff
@@ -0,0 +1,157 @@
+---
+title: 'Windows games on ChromeOS with Proton'
+metadesc: 'Discover how Proton and Steam make it possible to play Windows games on ChromeOS.'
+tags:
+  - games
+  - technical
+
+authors:
+  - renatopereyra
+date: 2023-01-03
+---
+
+[Steam is now available on ChromeOS](https://www.chromium.org/chromium-os/steam-on-chromeos/)
+and many users are already enjoying some of their favorite games. But you may be
+wondering how the magic happens.
+[Borealis is a VM running Arch Linux](https://chromeos.dev/en/posts/bringing-steam-to-chromeos),
+so Linux titles run the same way that they run on any Linux distribution. For
+Windows games, the story is more complex. Windows games run through a
+compatibility layer called Proton, which Steam downloads automatically when a
+user installs a Windows game on a Linux system.
+
+# So…what exactly is Proton?
```

Comment:
```suggestion
# So… what exactly is Proton?
```

This may be my personal preference, so please LMK, but it looks weird to me that there's no space after the ellipsis here, and it makes for a bad link (so and what get combined to `sowhat`). Thoughts on including the space?